src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/casedocuments/CaseSummariesHandler.java

```diff
@@ -19,8 +19,7 @@ protected boolean canHandleDocument(UploadCaseDocument uploadCaseDocument) {
 
         CaseDocumentType caseDocumentType = uploadCaseDocument.getCaseDocumentType();
         return uploadCaseDocument.getCaseDocumentFdr().equals(YesOrNo.NO)
-            && (caseDocumentType.equals(CaseDocumentType.POSITION_STATEMENT)
-            || caseDocumentType.equals(CaseDocumentType.SKELETON_ARGUMENT)
+            && (caseDocumentType.equals(CaseDocumentType.POSITION_STATEMENT_SKELETON_ARGUMENT)
```

Comment:
It's good practice to put the constant first to avoid NPE

Reply:
Although I agree with the principle, there are nuances here, for starters this is not new code and case document type is mandatory, so it will never be null.